internal/manifests/collector/parser/processor/processor_k8sattributes.go

```diff
@@ -62,6 +62,10 @@ func (o *K8sAttributesParser) GetRBACRules() []rbacv1.PolicyRule {
 			Resources: []string{"replicasets"},
 			Verbs:     []string{"get", "watch", "list"},
 		},
+		{
+			NonResourceURLs: []string{"/metrics"},
```

Comment:
> By adding this non-resource URL to the RBAC, it becomes possible for an OTEL collector to scrape authenticated endpoints such as control plane components.

@thefirstofthe300 this file creates RBAC for the k8sattribute processor.  This does not seem to be related to metrics

Reply:
or maybe I am missing something here